e2e/specs/clinical-forms.spec.ts

```diff
@@ -71,7 +71,7 @@ test('Fill a clinical form', async ({ page }) => {
     await chartPage.page.getByRole('button', { name: /save and close/i }).click();
   });
 
-  await test.step('Then I should see a success notification', async () => {
+  await test.step('Then I should see a success toast notification', async () => {
```

Comment:
It's a [snackbar](https://zeroheight.com/23a080e38/p/683580-notifications), not a toast.